spacy/errors.py

```diff
@@ -919,6 +919,8 @@ class Errors(metaclass=ErrorsWithCodes):
     E1035 = ("Token index {i} out of bounds ({length})")
     E1036 = ("Cannot index into NoneNode")
     E1037 = ("Invalid attribute value '{attr}'.")
+    E1038 = ("Function was called with `{arg1}`={arg1_values} and `{arg2}`={arg2_values} but that information is "
+             "conflicting.")
```

Comment:
Still not 100% happy with the phrasing, but:

```suggestion
    E1038 = ("Function was called with `{arg1}`={arg1_values} and "
             "`{arg2}`={arg2_values} but these arguments are conflicting.")
```